README.md

```diff
@@ -47,7 +47,8 @@ For Ginkgo core library:
 
 The Ginkgo CUDA module has the following __additional__ requirements:
 
-*   _CUDA 9.2+_ or _NVHPC Package 22.7+_
+*   _cmake 3.18+_ (If CUDA was installed through the NVIDIA HPC Toolkit, we require _cmake 3.22+_)
+*   _CUDA 10.1+_ or _NVHPC Package 22.7+_
```

Comment:
10.0 or 10.1?

Reply:
Good question, I forgot that 10.0 is mostly unused as 10.1 was released just after. I guess I'll just update the changelog to 10.1, since we don't test 10.0?

Reply:
I think so.